sdk/rust-examples/minigrep/README.markdown

```diff
@@ -0,0 +1,33 @@
+# A Simple mini version of grep, implemented using Veracruz freestanding execution engine (Case Sensitive, Only for ASCII)

```

Comment:
sp: Simple --> simple

sp: Case Sensitive --> case sensitive

sp: Only for ASCII --> only for ASCII